projects/components/src/table/header/table-header-cell-renderer.component.ts

```diff
@@ -18,25 +25,43 @@ import { TableColumnConfigExtended } from '../table.service';
       class="table-header-cell-renderer"
     >
       <ng-container *ngIf="this.columnConfig?.filterable && this.leftAlignFilterButton">
-        <ng-container *ngTemplateOutlet="filterButton"></ng-container>
+        <ng-container *ngTemplateOutlet="optionsButton"></ng-container>
       </ng-container>
-      <div class="title" [ngClass]="this.classes" (click)="this.sortChange.emit()">{{ this.columnConfig.title }}</div>
+      <div class="title" [ngClass]="this.classes" (click)="this.onSortChange()">{{ this.columnConfig.title }}</div>
       <ng-container *ngIf="this.columnConfig?.filterable && !this.leftAlignFilterButton">
-        <ng-container *ngTemplateOutlet="filterButton"></ng-container>
+        <ng-container *ngTemplateOutlet="optionsButton"></ng-container>
       </ng-container>
 
-      <ng-template #filterButton>
-        <ht-in-filter-button
-          class="filter-button"
-          [metadata]="this.metadata"
-          [attribute]="this.columnConfig.attribute"
-          [values]="this.columnConfig.filterValues"
-        ></ht-in-filter-button>
+      <ng-template #optionsButton>
+        <ht-popover class="options-button" [closeOnClick]="true">
+          <ht-popover-trigger>
+            <div #trigger>
+              <ht-icon icon="${IconType.MoreHorizontal}" size="${IconSize.Small}"></ht-icon>
+            </div>
+          </ht-popover-trigger>
+          <ht-popover-content>
+            <div [style.min-width.px]="trigger.offsetWidth" class="popover-content">
+              <div class="popover-item" (click)="this.onFilterValues()" *ngIf="this.isFilterable">Filter Values</div>
+              <div class="popover-item-divider" *ngIf="this.isFilterable"></div>
+              <div class="popover-item" (click)="this.onSortChange(SORT_ASC)">
```

Comment:
Are these alternate ways of sorting? Also, our sort is a tri-state right now, but this only shows two options (one of which may already be applied) with no way of unsetting the sorting.

Reply:
Correct. These are alternatives. Clicking on the header cycle sorting as usual.

Reply:
Should there be a third option to remove sort to match the tristate?